CHANGELOG.md

```diff
@@ -69,6 +70,7 @@ Here is a template for new release sections
 - axioms of hydro energy (incl. subclasses), wind energy, study, study report and model calculation (#871)
 - hydrogen turbine (#877)
 - has objective (#878)
+- typical day / year (#882)
```

Comment:
```suggestion

```

As the addition and the change of typical day and typical year are within one release I suggest to mention it only once.

Reply:
So change line 49 to:
`- typical period and subclasses (#875, #882)`